retis/src/module/ovs/bpf/kernel_exec_tp.bpf.c

```diff
@@ -195,6 +205,25 @@ DEFINE_HOOK_RAW(
 		}
 		break;
 		}
+	case OVS_ACTION_ATTR_DROP:
+		{
+		if (OVS_ACTION_ATTR_DROP >= config->action_max) {
```

Comment:
Isn't `bpf_core_enum_value_exists()` helpful here (also simplifying the config side), WDYT?

Also a nit: some c99 comments slipped in (like the ones below).

Reply:
`bpf_core_enum_value_exists` will not help as the value may exist. But  `bpf_core_enum_value_exists` might. Thanks for the pointer, I had completely forgotten about those helpsers